main.go

```diff
@@ -192,19 +182,17 @@ func runDns() {
 	// start DNS UDP serverTls
 	if *bindDnsOverTls {
 		go func() {
-
-			cert, key, err := GenerateSelfSignedCertKey(*publicIP, nil, nil)
+			dir := "/etc/"
```

Comment:
not a big fan of generating a certificate on run and put it on /etc/ directory. Maybe a temp directory is the way to go? or maybe we should just add the option to provide the certificate path and if it's not available, generate one?

Reply:
It's up to you. The /etc/ directory is for my convenience. Alternatively, you can create a directory similar to /usr/local/sniproxy